### ISSUE TITLE
Allow passing through CashScript contract options

### ISSUE THREAD

**Issue by 2qx** [mainnet-cash/mainnet-js#188]
Currently mainnet/contract defaults to p2sh20, but the default for CashScript is p2sh32.  

Users should be able to pass through any/all options as if it were CS directly.

**Comment by mainnet-pat** [comment]
I have a better proposal - to ditch the cashscript support from mainnet-js. The package is mature enough to be used standalone, there seems to be little benefit of using the mainnet-js wrapper over using cashscript directly atm.

**Comment by mr-zwets** [comment]
> I whole-heatedly support dropping the CashScript SDK whereever it's being used as a dependency.
> 
> And the whole @mainnet/contract package in this case.

I agree that dropping the @mainnet/contract package fully seems like the best way forward because of the arguments i outlined in #230
